crates/driver/src/infra/blockchain/contracts.rs

```diff
@@ -154,8 +170,10 @@ impl Contracts {
         &self.cow_amm_registry
     }
 
-    pub fn flashloan_wrappers(&self) -> &[contracts::IFlashLoanSolverWrapper] {
-        &self.flashloan_wrappers
+    pub fn flashloan_wrapper_by_lender(
```

Comment:
nit: Might be slightly nicer to already have the lookup happen in this function.